app/api/views/dashboards.py

```diff
@@ -50,3 +50,136 @@ def seller_dashboard():
     supplier_responses = briefs.get_supplier_responses(current_user.supplier_code)
 
     return jsonify(items=supplier_responses, supplier={'name': supplier.name, 'code': supplier.code}), 200
+
+
+@api.route('/buyers/my-briefs', methods=['GET'])
```

Comment:
does `/dashboard/my/briefs` look better?

Reply:
I agree with the dashboard part - the rest you could argue either way

Reply:
Routes have been updated.